### ISSUE TITLE
Messages sent to anyone without a previous conversation are not stores

### ISSUE THREAD

**Issue by ADepic** [moezbhatti/qksms#1324]
### DESCRIPTION
**Device:** Huawei Honor View 10
**OS:**EMUI 8.0.0 (Android 8.0.0)
**Android Security Patch:** 1 August 2018

When you send a message to anyone (whether they are a contact/not a contact) who you have not had a conversation with, the message you sent is not stored. The recipient recieves the message.

[Wait let me rephrase that so it makes more sense]

If you try to start a new conversation by sending a message, the recipient receives the message but the new conversation is not stored.

The is a quite a urgent bug, since it means you can't keep track of messages unless the recipient messages you back. Please could you fix it quickly?

### STEPS
1. Press the "Plus" button to start new conversation.
2. Find someone you have never sent a message to
3.  Send a message
4. Press the back button
5. The message is not stored.

### EXPECTED

The message you just sent is recieved, but a copy of the message is not stored on your phone. The only way to store a conversation on your phone is if the other user messages you back.

### OBSERVATIONS

Messages you send to new conversations are not stored.

Furthermore, you can't access the settings (to block the contact, etc.). When you click on the three dots in the top right hand corner, the settings slides into view for half a second and then immediately slides out of view again.

**Comment by prg318** [comment]
potentially related? https://github.com/moezbhatti/qksms/issues/1296

**Comment by AdelineDek** [comment]
I have exactly the same problem with Huawei P10 (same OS and security patch)

It seems the new message is not stored in the Android database : it doesn't appear using the native application.

**Comment by georgkrause** [comment]
I have the same problem with Android 8.1.0 and EMUI 8.1.0 on Honor 10 with Patch from 1.9.

**Comment by moezbhatti** [comment]
Fixed in https://github.com/moezbhatti/qksms/commit/e610a373ca5416bc5ed5a648122a20b6e56e3727
